backend/imports/tasks/methods.js

```diff
@@ -35,13 +35,13 @@ export const getMyTasks = function () {
  * @param text {string}
  * @return {string} inserted document _id
  */
-export const insertTask = function ({ text }) {
-  const userId = this.userId
-  checkUser(userId)
-  const checked = false
-  const createdAt = new Date()
-  return TasksCollection.insert({ text, userId, checked, createdAt })
-}
+export const insertTask = async function ({ text }) {
+  const userId = this.userId;
+  checkUser(userId);
+  const checked = false;
+  const createdAt = new Date();
+  return await TasksCollection.insert({ text, userId, checked, createdAt });
```

Comment:
shouldn't it be `await TasksCollection.insertAsync` ?

Reply:
Yes you are right